libs/community/langchain_community/vectorstores/azuresearch.py

```diff
@@ -431,16 +431,16 @@ def vector_search_with_score(
         Returns:
             List of Documents most similar to the query and score for each
         """
-        from azure.search.documents.models import Vector
+        from azure.search.documents.models import VectorizedQuery
 
         results = self.client.search(
             search_text="",
-            vectors=[
-                Vector(
+            vector_queries=[
+                VectorizedQuery(
                     value=np.array(
```

Comment:
is `value` supposed to be `vector` now?

Reply:
@baskaryan You were right. I missed that to copy it over to my other repo. I've changed it.